pkg/kubelet/rkt/rkt.go

```diff
@@ -2054,13 +2060,10 @@ func (r *Runtime) GetPodStatus(uid types.UID, name, namespace string) (*kubecont
 		}
 	}
 
-	if latestPod != nil {
-		// Try to fill the IP info.
-		for _, n := range latestPod.Networks {
-			if n.Name == defaultNetworkName {
-				podStatus.IP = n.Ipv4
-			}
-		}
+	// TODO(euank): this will not work in host networking mode
+	containerID := kubecontainer.ContainerID{ID: string(uid)}
```

Comment:
It looks like it actually does get set there, yeah. Maybe this should be a `note`, not a `TODO`, I'll do some quick experimenting.